coalib/bearlib/aspects/Formatting.py

```diff
@@ -393,3 +394,6 @@ class docs:
     preferred_quotation = Taste[str](
         'Represents the preferred quotation',
         ('\'', '"'), default='\'')
+    force_preferred_quotation = Taste[bool](
```

Comment:
how is this different from `preferred_quotation`.

This needs a _lot_ of documentation to explain why it is different, and when it should be used, as compared to just `preferred_quotation`.

A real example would be very helpful.

Reply:
Perhaps part of the problem is what is `preferred_quotation`, and why isnt it mandatory to follow the preferred quotation?